examples/02-plot/anti-aliasing.py

```diff
@@ -0,0 +1,95 @@
+"""
+.. _antialiasing_example:
+
+Anti-Aliasing
+~~~~~~~~~~~~~
+
+PyVista supports three types of anti-aliasing:
+
+* ``"ssaa"`` - Super-Sample Anti-Aliasing
+* ``"mxaa"`` - Multi-Sample Anti-Aliasing
+* ``"fxaa"`` - Fast Approximate Anti-Aliasing
+
+By default, anti-aliasing is disabled, but can be enabled globally with:
+
+.. code:: python
+
+   >>> import pyvista as pv
+   >>> pv.global_theme.antialiasing = 'ssaa'
+
+**Which Anti Aliasing Technique should You use?**
+
+Those who have PCs with high-end configuration should opt for ``"ssaa"`` or
+``"mxaa"``. Low-end PCs should use ``"fxaa"``.
+
+"""
+
+import pyvista as pv
+from pyvista import examples
+
+bunny = examples.download_bunny()
+
+###############################################################################
+# No Anti-Aliasing
+# ~~~~~~~~~~~~~~~~
+# First, let's show a plot without any anti-aliasing.
+
+# obtained with `cpos = pl.show(return_cpos=True)`
+cpos = [(-0.08566, 0.18735, 0.20116), (-0.05332, 0.12168, -0.01215), (-0.00151, 0.95566, -0.29446)]
+
+pl = pv.Plotter()
+pl.add_mesh(bunny, show_edges=True)
+pl.disable_anti_aliasing()
+pl.camera_position = cpos
+pl.show()
+
+
+###############################################################################
+# Fast Approximate Anti-Aliasing (FXAA)
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Also known as Fast Approximate Anti-Aliasing, FXAA was crafted by
+# Nvidia. When it comes to anti-aliasing on low-end devices/PCs, FXAA is the
+# best technique. This is because, in terms of hardware or GPU, FXAA is not
+# that demanding. It directly smoothens the 2D image as soon as it appears on
+# the screen. This reduces the strain over GPU, making it best for low-end PCs.
```

Comment:
I didn't understand "directly smoothens the 2D image as soon as it appears on the screen" at all, first. Having looked at the rest of the PR I think this means that it only blurs a rendered 2d image at the end, but this was not clear to me at all.

Reply:
I’ve cleaned up the docs.